companion-select-text-insert/selected-text-below.qml

```diff
@@ -0,0 +1,103 @@
+import QtQml 2.0
+import QOwnNotesTypes 1.0
+
+Script {
+    /**
+     * Writes the selected text on the webpage that was sent to QOwnNotes from
+     * the browser extension into the current note
+     */
+ 
+    property string linkStyle;
+
+    // register your settings variables so the user can set them in the script settings
+    //
+    // unfortunately there is no QVariantHash in Qt, we only can use
+    // QVariantMap (that has no arbitrary ordering) or QVariantList (which at
+    // least can be ordered arbitrarily)
+    property variant settingsVariables: [
+        {
+            "identifier": "linkStyle",
+            "name": "Insert Link as Ref Style",
+            "description": "Insert link As Reference , Below or NoLink ?",
```

Comment:
s/Insert link As Reference , Below or NoLink ?/Insert link As Reference, Below or NoLink?/
And can you please explain what those settings mean in the description of info.json.

Reply:
that description cannot understand new lines? i tried \n and it ignores

Reply:
Try `<br>`, if that's not working then there are no newlines.

Reply:
gonna try br

Reply:
<br> works Thanks ,